scripts/build.py

```diff
@@ -10,35 +10,44 @@
 from datetime import datetime
 from string import StringFileConverter
 
-PathVar = os.environ.get('Path')
-Paths = PathVar.split(';')
-PATH = ""
-for candidatePath in Paths:
-    if "devkitARM" in candidatePath:
-        PATH = candidatePath
-        break
-if PATH == "":
-	print('DevKit does not exist in your Path variable.\nChecking default location.')
-	PATH = 'C://devkitPro//devkitARM//bin'
-	if os.path.isdir(PATH) == False:
-		print("...\nDevkit not found.")
-		sys.exit(1)
-	else:
-		print("Devkit found.")
+if sys.platform == 'linux':
```

Comment:
added a case for linux. Yes, this part can definitely be improved and windows shouldn't be treated as a simple `else` case. I guess you will want to test and improve it on windows.